images/build/go-runner/go-runner.go

```diff
@@ -79,7 +80,11 @@ func configureAndRun() error {
 	cmd.Stderr = errStream
 
 	log.Printf("Running command:\n%v", cmdInfo(cmd))
-	err := cmd.Start()
+	if cmd.Stdout == os.Stdout && cmd.Stderr == os.Stderr {
+		err = syscall.Exec(cmd.Path, cmd.Args, syscall.Environ())
```

Comment:
Hm, this feels like a potentially breaking change. Shall we make it an optional feature or such?

Reply:
I agree, but the idea was to automatically/intelligently save systems resources by avoiding the parent process (go-runner) if not needed. What do you think about an opt-out argument/flag?